src/_packagedcode/pypi.py

```diff
@@ -684,10 +686,17 @@ def parse(cls, location):
 
         metadata = {}
         parser = ConfigParser()
+        dependent_packages = []
         with open(location) as f:
             parser.read_file(f)
-
         for section in parser.values():
+            if section.name == 'options':
```

Comment:
You likely want to add also the legacy "setup_requires" and "test_requires" as well as the "python_requires"

Reply:
@pombredanne can you provide me some sample setup.cfg file with legacy "setup_requires", "test_requires", "python_requires"